From 8370da992c752e4238de7a8b02ce0e179353c959 Mon Sep 17 00:00:00 2001
From: Alexander Egorenkov <egorenar-dev@posteo.net>
Date: Sun, 6 Sep 2020 10:37:41 +0200
Subject: [PATCH] Makefile.inc: Enable systemd if ENABLE_SYSTEMD is defined

systemd is disabled by default, to activate it set ENABLE_SYSTEMD.

Signed-off-by: Alexander Egorenkov <egorenar-dev@posteo.net>
---
 Makefile.inc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Makefile.inc b/Makefile.inc
index d4d1e0dd..32bc1846 100644
--- a/Makefile.inc
+++ b/Makefile.inc
@@ -35,6 +35,7 @@ ifndef RUN
 	endif
 endif
 
+ifdef ENABLE_SYSTEMD
 ifndef SYSTEMD
 	ifeq ($(shell pkg-config --modversion libsystemd >/dev/null 2>&1 && echo 1), 1)
 		SYSTEMD = $(shell pkg-config --modversion libsystemd)
@@ -45,6 +46,7 @@ ifndef SYSTEMD
 		endif
 	endif
 endif
+endif
 
 ifndef SYSTEMDPATH
 	SYSTEMDPATH=usr/lib
-- 
2.28.0

